### DISCUSSION TITLE
Is there a report to see income, outcome and budgets in relation

### DISCUSSION THREAD

**Discussion by BennyLi** [firefly-iii/firefly-iii#4954] [General]
I am running Firefly III version 5.4.6.

**Description**
I'm looking for a way to see my incomes in relation to the spent money and the open budgets, so I can quickly determine if all budgets are realistic anymore. I've checked the various reports, but none fits what I'm looking for. Something like a little box on the frontpage showing the sum of all income, the sum off all spent money, the sum of all defined budgets (the amount of it, not was is already spent) and the the sum of these three together would be enough for me, I think ... 🙄

Is there already some way to accomplish this?

**Bonus points**

- [x] I have read the FAQ at https://bit.ly/FF3-FAQ
- I added a screenshot
- I added log files
- I was able to replicate the issue on the demo site.

**Comment by JC5** [comment]
I use the default report, and then check each budget individually. Is that not something you could use?
